admin-dev/themes/new-theme/scss/config/_keyframes.scss

```diff
@@ -1,17 +1,11 @@
 // EMPTY STATES
 @keyframes showcase-img-appearance {
   from {
-    -webkit-transform: translate(-20px);
-    -ms-transform: translate(-20px);
-    -webkit-transform: translate(-20px);
-    transform: translate(-20px);
     opacity: 0;
- }
+    transform: translate(-20px);
```

Comment:
So `transform` is automatically compiled but `transition` (for example) requires a `@include transition()`

Reply:
Bootstrap doesn't return an error on using transform and it looks like it has no mixins for it